packages/perseus/src/widgets/number-line/number-line-validator.test.ts

```diff
@@ -0,0 +1,109 @@
+import numberLineValidator from "./number-line-validator";
+
+import type {
+    PerseusNumberLineRubric,
+    PerseusNumberLineUserInput,
+} from "../../validation.types";
+
+const baseInput: PerseusNumberLineUserInput = {
+    isTickCrtl: true,
+    numLinePosition: 1,
+    rel: "eq",
+    numDivisions: 10,
+    divisionRange: [-10, 10],
+};
+
+const baseRubric: PerseusNumberLineRubric = {
+    correctRel: "eq",
+    correctX: -1.5,
+    divisionRange: [1, 12],
+    initialX: -1,
+    labelRange: [null, null],
+    labelStyle: "decimal",
+    labelTicks: true,
+    numDivisions: null,
+    range: [-1.5, 1.5],
+    snapDivisions: 2,
+    static: false,
+    tickStep: 0.5,
+    isInequality: false,
+};
+
+function generateInput(
+    extend?: Partial<PerseusNumberLineUserInput>,
+): PerseusNumberLineUserInput {
+    return {...baseInput, ...extend};
+}
+
+function generateRubric(
+    extend?: Partial<PerseusNumberLineRubric>,
+): PerseusNumberLineRubric {
+    return {...baseRubric, ...extend};
+}
+
+describe("numberLineValidator", () => {
+    it(`is invalid when outside allowed range`, () => {
```

Comment:
Nit: Should the backticks/"`" be regular quotation marks in the tests?

Reply:
Probably isn't a big deal either way. This is just an old weird habit of mine to use backticks in tests. Switched to quotes to be more consistent with our other tests.